### ISSUE TITLE
How can we get OOD 2.0.19 deployed using ood-ansible ?

### ISSUE THREAD

**Issue by xpillons** [OSC/ood-ansible#130]
today if I run these playbooks the release 2.0.18 is deployed, which is not the latest. How can I specify a version ? or is there a way to retrieve the public RPMs for OOD ?

**Comment by xpillons** [comment]
Thank you that would be great.

**Comment by xpillons** [comment]
Actually I don't know where to pickup the 2.0.20 RPM as it's not listed here https://yum.osc.edu/ondemand/2.0/web/el7/x86_64/ the latest being 2.0.18

**Comment by johrstrom** [comment]
You have to pull it from our `latest` repo.  We just published it and we haven't verified it yet. (I knew 19 wouldn't make it to general availability because we still hadn't fixed that file upload issue)

I'm about to pull in the PR attached to this and release it. Here are the configs to get 20 from latest.

```
ondemand_package: 'ondemand-2.0.20'
ondemand_package_excludes:
  - '*-2.1'
rpm_repo_url: 'https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest-1-6.noarch.rpm'
```

**Comment by johrstrom** [comment]
2.0.20 is now available on the 2.0 repo so you don't need to use the latest now.
